### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by bhtibrewal**
### Features added: 
- Navbar
- Homepage

### Files to Check:
- [App.jsx](src/App.jsx)
- [Button.jsx](src/components/Button.jsx)
- [Navbar.jsx](src/components/navbar/Navbar.jsx)
- [HomePage.jsx](src/pages/home_page/HomePage.jsx)

image
image

### CODE REVIEW

File: src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      <i class="fas fa-search"></i>
+      <input placeholder="Search..." type="text" />
+    </div>
+  );
+};
+export const RightNav = () => {
+  return (
+    <div class="right-side">
+      <div onclick="openLogin()">
+        <i class="fa-regular fa-user"></i>
```
Comment (Ankur9669) [inline src/components/navbar/Navbar.jsx]: fix: onclick should be replaced by onClick and i think function could be removed

File: src/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <button className={`btn btn-primary ${className}`} onClick={onClick}>
+      {children}
+    </button>
+  );
+};
+export const OutlineButtonPrimary = ({ children, className, onClick }) => {
+  return (
+    <button class={`btn outline-btn-primary ${className}`} onClick={onClick}>
+      {children}
+    </button>
```
Comment (Ankur9669) [inline src/components/Button.jsx]: fix: class should be replaced by className

File: src/components/Button.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  );
+};
+export const OutlineButtonSecondary = ({ children, className, onClick }) => {
+  return (
+    <button class={`btn outline-btn-secondary ${className}`} onClick={onClick}>
+      {children}
```
Comment (Ankur9669) [inline src/components/Button.jsx]: fix: class should be replaced by className

File: src/components/navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import "./navbar.css";
+
+export const LeftNav = () => {
+  return (
+    <a href="/">
+      <h3>
+        <i class="fa-brands fa-shopify"></i>Artsy Shop
+      </h3>
```
Comment (Ankur9669) [inline src/components/navbar/Navbar.jsx]: fix: class should be replaced by className

File: src/components/navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    </a>
+  );
+};
+export const Search = () => {
+  return (
+    <div class="artsy-input icon-input">
+      <i class="fas fa-search"></i>
+      <input placeholder="Search..." type="text" />
```
Comment (Ankur9669) [inline src/components/navbar/Navbar.jsx]: fix: class should be replaced by className

File: src/components/navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+export const Navbar = (props) => {
+  return (
+    <header class="arstyShop-header header">
+      {props.leftNav}
+      {props.search}
+      {props.rightNav}
+    </header>
+  );
+};
+
```
Comment (Ankur9669) [inline src/components/navbar/Navbar.jsx:12]: fix: classes should be replaced by className

File: src/pages/home_page/HomePage.jsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
+import "./home_page.css";
+import collection_img from "./../../assets/images/collection_item_img.jpeg"
+import { ButtonPrimary } from "../../components/Button";
+
+const Categories = ({ link, category, img_url }) => {
+  return (
+    <a href={link}>
+      <div class="categories">
+        <img src={img_url} />
+        <button class="btn category-name">{category}</button>
```
Comment (Ankur9669) [inline src/pages/home_page/HomePage.jsx]: fix: alt tag is missing
fix: class should be replaced by className

File: src/pages/home_page/HomePage.jsx [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+      <div class="collection-text">
+        <span class="btn collection-tag">{tag}</span>
+        <div>
+          <h3>{heading}</h3>
+          <p>{subtitle}</p>
+        </div>
+      </div>
+      <div class="collection-artworks flex-col">
+        {item.map((i) => {
+          return (
```
Comment (Ankur9669) [inline src/pages/home_page/HomePage.jsx]: fix: key attribute is missing

File: src/pages/home_page/HomePage.jsx [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+            <div class="collection-artwork-item">
+              <img src={img} />
+            </div>
```
Comment (Ankur9669) [inline src/pages/home_page/HomePage.jsx]: fix: alt tag is missing

File: src/pages/home_page/HomePage.jsx [hunk 10]
```diff
@@ -0,0 +1,123 @@
+import "./home_page.css";
+import collection_img from "./../../assets/images/collection_item_img.jpeg"
+import { ButtonPrimary } from "../../components/Button";
+
+const Categories = ({ link, category, img_url }) => {
+  return (
+    <a href={link}>
+      <div class="categories">
+        <img src={img_url} />
```
Comment (sri-rishi) [inline src/pages/home_page/HomePage.jsx]: Suggestion: provide "alt" in < img /> tag, it will help in accessibility

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image

UI Suggestion: moving numbers little bit up relative to icons will look better

**Review by sri-rishi — COMMENTED** [review]
UI and code looks good just some suggestion which can be done,  apart form that nice wok

File: src/components/Button.jsx [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+export const OutlineButtonPrimary = ({ children, className, onClick }) => {
+  return (
+    <button class={`btn outline-btn-primary ${className}`} onClick={onClick}>
+      {children}
+    </button>
+  );
+};
+export const OutlineButtonSecondary = ({ children, className, onClick }) => {
+  return (
+    <button class={`btn outline-btn-secondary ${className}`} onClick={onClick}>
```
Comment (yaswanthmyneni) [inline src/components/Button.jsx]: changes: use className here.

File: src/components/navbar/Navbar.jsx [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+      </a>
+
+      <a class="cart" href="/pages/cart/cart.html">
+        <i class="fa-solid fa-bag-shopping"></i>
+        <span class="items">25</span>
+      </a>
+    </div>
+  );
+};
+export const Navbar = (props) => {
```
Comment (yaswanthmyneni) [inline src/components/navbar/Navbar.jsx:3]: changes: destructure instead of writing props

File: src/pages/home_page/HomePage.jsx [hunk 13]
```diff
@@ -0,0 +1,123 @@
...
+    <a href="" class="collection-items">
+      <div class="collection-image">
+        <img alt="collection image " class="collection-image" src={bg_img} />
+      </div>
+      {children}
+    </a>
+  );
+};
+export const HomePage = () => {
+
```
Comment (yaswanthmyneni) [inline src/pages/home_page/HomePage.jsx:6]: suggestion: keep individual components in separate file.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes.

File: src/pages/home_page/HomePage.jsx [hunk 14]
```diff
@@ -0,0 +1,123 @@
...
+                category={i}
+                img_url={
+                  "https://cdn.singulart.com/artworks/v2/cropped/3813/main/carousel/1202203_f053297adb8623bd3c615562c7521b67.jpeg"
+                }
+              />
+            );
+          })}
+        </div>
+      </section>
+
```
Comment (Ankur9669) [inline src/pages/home_page/HomePage.jsx:28]: fix: change class to className

**Review by Ankur9669 — APPROVED** [review]
The code looks good, but try to make small PR's, it would be easy to review them

File: src/pages/home_page/HomePage.jsx (hunk 10 above)
Comment (bhtibrewal) [reply src/pages/home_page/HomePage.jsx]: ok i will add that, thank you for pointing out

**Pull request merged**